---
geometry: margin=1in
month: "`r format(Sys.Date(), '%B')`"
year: "`r format(Sys.Date(), '%Y')`"
params:
    model: 2
preamble: |
output:
  sa4ss::techreport_pdf:
    default
  bookdown::pdf_document2:
    keep_tex: true
    keep_md: true
header-includes:
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
  - \usepackage{placeins}
lang: en
papersize: a4
---

```{r opts, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
knitr::knit_hooks$set(plot = function(x,options) {
      base = knitr::opts_knit$get('base.url')
      if (is.null(base)) base = ''
      alt = ifelse (is.null(options$alt),"",options$alt)
      cap = ifelse (is.null(options$caption),"",options$caption)
      if (alt != ""){
        sprintf('![%s](%s%s "%s")', cap, base, x, alt)
      } else {
        sprintf('![%s](%s%s)', cap, base, x)
        }
  })

source("C:/Stock_Assessments/VRML_Assessment_2021/GitHub/Vermilion_2021/code/dir_recent.R")
dir.path <- ifelse(params$model=="north",
             "C:/Stock_Assessments/VRML_Assessment_2021/Model_files/NCA/",
             "C:/Stock_Assessments/VRML_Assessment_2021/Model_files/SCA/")
model.dir <- dir_recent(dir = dir.path, pattern = "Verm")
base_loc <- paste0(model.dir)


if (is.null(params$model) & file.exists("00mod.Rdata")) {
  stopifnot(file.exists("00mod.Rdata"))
  load("00mod.Rdata")
} else {
  load(dir(base_loc, pattern = "00mod.Rdata", full.names = TRUE))
}


spp = 'vermilion rockfish'
Spp = 'Vermilion rockfish'
fig_loc = paste0(getwd(), "/figures")
table_loc = paste0(getwd(), "/tables")
# model_loc = "C:/Stock_Assessments/VRML_Assessment_2021/Model_files/"

model.area <- ifelse(params$model=="north", "NCA", "SCA")

if(params$model=="north"){
model$FleetNames_parsed <- c("commercial hook-and-line fishery",
                           "commercial trawl fishery",
                           "commercial net fishery",
                           "recreational PC retained fishery",
                           "recreational PC discard fishery",
                           "recreational PR retained fishery",
                           "recreational PR discard fishery",
                           "Deb Wilson-Vandenberg onboard survey",
                           "West coast groundfish bottomfish trawl survey",
                           "recreational PC onboard survey",
                           "Abrams thesis research survey",
                           "SWFSC groundfish ecology survey",
                           "California Collaborative Fisheries Research Project survey")
} else {
model$FleetNames_parsed <- c("commercial hook-and-line fishery",
                           "commercial trawl fishery",
                           "commercial net fishery",
                           "recreational PC retained fishery",
                           "recreational PC discard fishery",
                           "recreational PR retained fishery",
                           "recreational PR discard fishery",
                           "NWFSC hook-and-line survey",
                           "West Coast Groundfish Bottomfish Trawl Survey",
                           "recreational PC onboard survey",
                           "CDFW reserach (aka green binder)")
}

model$FleetNames_label <- stringr::str_replace_all(model$FleetNames, "_", "-")



```

<!--chapter:end:00a.Rmd-->

---
author:
  - name: Melissa H. Monk
    code: 1
    first: M
    middle: H
    family: Monk
  - name: E. J. Dick
    code: 1
    first: E
    middle: J
    family: Dick
  - name: John C. Field
    code: 1
    first: J
    middle: C
    family: Field
  - name: Emma M. Saas
    code: 1
    first: E
    middle: M
    family: Saas
author_list: Monk, M.H., E.J. Dick, J.C. Field, E.M. Saas
affiliation:
  - code: 1
    address: Southwest Fisheries Science Center, U.S. Department of Commerce, National
      Oceanic and Atmospheric Administration, National Marine Fisheries Service, 110
      McAllister Way, Santa Cruz, California 95060
address: ^1^Southwest Fisheries Science Center, U.S. Department of Commerce, National
  Oceanic and Atmospheric Administration, National Marine Fisheries Service, 110 McAllister
  Way, Santa Cruz, California 95060
---

<!--chapter:end:00authorsnorth.Rmd-->

---
bibliography:
  - sa4ss.bib
---

<!--chapter:end:00bibliography.Rmd-->

---
title: The status of Vermilion Rockfish (_Sebastes miniatus_) and Sunset Rockfish (_Sebastes crocotulus_) in U.S. waters off the coast of California north of Pt. Conception in 2021
---

<!--chapter:end:00titlenorth.Rmd-->

\pagebreak
\pagenumbering{roman}
\setcounter{page}{1}

\renewcommand{\thetable}{\roman{table}}
\renewcommand{\thefigure}{\roman{figure}}


\setlength\parskip{0.5em plus 0.1em minus 0.2em}

<!--chapter:end:01a.Rmd-->

```{r executive, echo = FALSE}
executive <- list()
executive[["stock"]] <- paste0("This assessment reports the status of the 
Vermilion Rockfish (_Sebastes miniatus_) and Sunset Rockfish 
(_Sebastes crocotulus_) in U.S. waters off the coast of California south
  of Pt. Conception through 2020.")
```
# Executive Summary{-}
To be completed after the STAR panel.

## Stock{-}
This assessment reports the status of the vermlion rockfish (_Sebastes miniatus_) 
and sunset rockfish (_Sebastes crocotulus_) complex (referred to as vermilion 
throughout), resource in U.S. waters off the coast of California `r params$model` 
of Point Conception ($34^\circ 27^\prime$ N. latitude) using data 
through 2020.


## Landings{-}
Replace text.

## Data and Assessment{-}
Replace text.

## Stock Biomass{-}
Replace text.

## Recruitment{-}
Replace text.

## Exploitation Status{-}
Replace text.

## Reference Points{-}
Replace text.

## Management Performance{-}
Replace text.

## Unresolved Problems and Major Uncertainties{-}
Replace text.

## Decision Table{-}
Replace text.

## Research and Data Needs{-}
Replace text.

<!--chapter:end:01executive.Rmd-->

\pagebreak
\setlength{\parskip}{5mm plus1mm minus1mm}
\pagenumbering{arabic}
\setcounter{page}{1}
\renewcommand{\thefigure}{\arabic{figure}}
\renewcommand{\thetable}{\arabic{table}}
\setcounter{table}{0}
\setcounter{figure}{0}

<!--chapter:end:10a.Rmd-->

# Introduction

## Basic Information
`r executive[["stock"]]`

##Basic Information and Life History

*Population Structure and Complex Assessment Considerations*

Provided by John Harms (NWFSC)

A group of researchers from the NWFSC and SWFSC is collaborating on a project to genotype tissue specimens collected from the vermilion and sunset rockfish cryptic pair captured during the West Coast Groundfish Bottom Trawl Survey and the Southern CA Shelf Rockfish Hook and Line Survey for the years 2004 - 2019.  Funding for this project was obtained through the Saltonstall-Kennedy program for FY 2020 through a proposal led by representatives from Pacific States Marine Fisheries Commission and the commercial passenger fishing vessel industry in southern California.  

After combining with specimens obtained through other collection efforts along the West Coast, approximately 25,000 tissue specimens will be analyzed.  Some earlier efforts to separate this cryptic pair to species used mitochondrial DNA (mtDNA) markers.  However, due to a one-way mitochondrial introgression from the vermilion genome into the sunset genome, a portion of the sunset rockfish population contains mitochondrial DNA sequences consistent with vermilion rockfish resulting in incorrect species assignments for these introgressed individuals during the prior research project.  The current research has identified a robust suite of genetic markers within the nuclear genomes of the two species that definitively separates vermilion and sunset rockfish (including  introgressed sunset rockfish), canary rockfish, first generation vermilion-sunset hybrids, and identifies emerging patterns of intraspecific stock structure within the two sister species.

Once the collected specimens have been genotyped, any species-specific differences in spatial and depth distribution, size composition, weight-length relationships, and other biological characteristics will be identified.  Using previously collected otoliths and ovaries, the demographics of the two species including age and growth and reproductive biology parameters such as length and age at 50\% maturity and the prevalence of skip spawning will be explored and compared.  These new genotyping results will be combined with data from the prior mtDNA work to  evaluate whether introgressed sunset rockfish represent a biologically intermediate subform of the species complex.  The effort also proposes to develop and test the efficacy of models to predict the relative proportion of the two species based upon explanatory variables including latitude, depth, species of co-occurrence, oceanographic parameters, habitat descriptors and/or other information.  The anticipated completion of the genotyping of all specimens is approximately December 2021 with provision of final results by the end of FY 2022.

This research is aimed at providing information to support the successful stock assessment of this commercially and recreationally valuable cryptic species pair and is responsive to any data gaps identified by the assessment community.  If successful, this research, conducted in close communication with stock assessors, may also assist the PFMC in establishing best practices for the assessment and management of cryptic species complexes.  Though this project will only focus on nominal vermilion rockfish specimens collected through the 2019 survey field season, it may be advisable that tissue specimens collected aboard fishery-independent surveys as well as through fishery-dependent programs continue to be genotyped on an ongoing basis to support continued and timely monitoring of this economically and ecologically important species complex.


## Early Life History


##Map


##Ecosystem Considerations





<!--chapter:end:11a-introduction-bio.Rmd-->

## Historical and Current Fishery Information
Replace text.

## Summary of Management History and Performance
Replace text.



##Management Performance

## Foreign Fisheries
Replace text.



<!--chapter:end:11b-introduction-fishery.Rmd-->

# Data
A description of each data source is provided below (Figure).

<!--chapter:end:20-data.Rmd-->

## Biological Data


### Length and Age Compositions

Differences in length compositions and length-at-age were explored 
north and south of Pt. Conception prior to any modelling. There are 
currently no available data to separate sunset and vermilion rockfishes.


Length compositions were provided from the following sources:

*North of Pt. Conception*

* Commercial sources
  + CALCOM (1978-2020)
* Recreational sources
  + Miller and Gotshall dockside survey (1959-1960)
  + CPFV samples from Commercial port samplers (1978-1979)
  + Deb Wilson-Vandenberg's onboard observer survey (1988-1998)
  + MRFSS dockside survey (1980-2003)
  + CRFS onboard and dockside survey (2004-2019)
* Fishery-independent surveys
  + CCFRP hook-and-line survey (2007-2018)
  + West Coast Groundfish Bottown Trawl Survey (2003-2019)



*South of Pt. Conception*

* Commercial sources
  + CALCOM (1978-2020)
* Recreational sources
  + Ally et al. onboard observer survey (1986-1989)
  + Collins and Crooke onboard observer survey (1975-1978)
  + MRFSS dockside survey (1980-2003)
  + CRFS onboard and dockside survey (2004-2018)
* Fishery-independent surveys
  + NWFSC Hook-and-Line Survey (2004-2019)
  + West Coast Groundfish Bottown Trawl Survey (2003-2019)



The length composition of all fisheries aggregated across time by fleet is in Figure
\ref{fig:comp_lendat_aggregated_across_time} and Table xxx. 
Descriptions and details of the length composition data are in the above section 
for each fleet or survey.

### Age Structures


**External Fits to Growth**


Fits to the von Bertalanffy growth curve, 
$L_i = L_{\infty}e^{(-k[t-t_0])}$, where $L_i$ is the length (cm) at age $i$, $t$ is age in 
years, $k$ is rate of increase in growth, $t_0$ is the intercept, and $L_{\infty}$ 
is the asymptotic length, were explore by species and sex. 



### Ageing Precision and Bias

Uncertainty in ageing error was estimated using a collection of 357 `r spp` 
otoliths with two age reads between the NWFSC (reader 1, B. Kamikawa) and the 
SWFSC (reader 2, D. Watters) (Figure \@ref(fig:reader1reader2)).
Age-composition data used in the model were from a number of sources described 
above. The same readers aged otoliths for both `r spp` stock assessmetnt models. 
Age reader 1 read all of the otoliths for the southern model and both readers read 
otoliths for the northern California model. In addition to the otoliths from these 
two regions, the same two readers aged fish for a Committee of Age Reading Experts 
(CARE)exchange among four ageing labs, initiated by the SWFSC.

Ageing error was estimated using publicly available software [@Thorson2012].
The software setting for bias was set to unbiased for reader 1 who was more 
experienced. The $\Delta AIC$ among the top three models was less than two. The 
best fitting model selected curvilinear bias for reader 1 and curvilinear standard 
deviation for both readers.  
An analysis of ageing error removing one fish aged at 88 by reader 1 and 78 by reader 2 
selected the the model with reader 2 as unbiased and curvilinear standard deviation 
(Figure \@ref(fig:oldfish)). The reading of the oldest aged fish falls within the 95\% confidence 
internal using this model (Figure \@ref(fig:truereads)).  
The latter model was selected for use in the assessment. 
 

The resulting estimate indicated a standard deviation in age readings 
increasing from 0.001 years at age 0 to a standard deviation of 2.37 years at age 70, 
the first year of the plus group in the assessment model.



### Maturation and Fecundity



### Natural Mortality

Natural mortality was not directly measured, so life-history based empirical relationships were used. The Natural Mortality Tool (NMT; https://github.com/shcaba/Natural-Mortality-Tool), a Shiny-based graphical user interface allowing for the application of a variety of natural mortality estimators based on measures such as longevity, size, age and growth, and maturity, was used to obtain estimates of natural mortality. The NMT currently provides 19 options, including the Hamel [-@Hamel2015] method, which is a corrected form of the Then et al. [-@Then2018] functional regression model and is a commomly applied method for west coast groundfish. The NMT also allows for the construction of a natural mortality prior weighted across methods by the user. 




### Sex Ratio

No information on the sex ratio at birth was available so it was assumed to be 50:50. 


### Length-Weight Relationship

The length(cm)-weight(kg) relationship for `r spp` was estimated outside the 
model using California biological data available from fishery-independent 
data sources. The estimated length-weight relationship for female fish was $W$=`r format(model$MGparmAdj$Wtlen_1_Fem[1], scientific = TRUE)`$L$^`r round(model$MGparmAdj$Wtlen_2_Fem[1],2)`^ and males at $W$=`r format(model$MGparmAdj$Wtlen_1_Mal[1], scientific = TRUE)`$L$^`r round(model$MGparmAdj$Wtlen_2_Mal[1], 2)`^ (Figure xx).



### Steepness

The Thorson-Dorn rockfish prior (developed for use West Coast rockfish assessments) conducted by James Thorson (personal communication, NWFSC, NOAA) and reviewed and endorsed by the Scientific and Statistical Committee (SSC) in 2017, has been a primary source of information on steepness for rockfishes. This approach, however, was subsequently rejected for future analysis in 2019 when the new meta-analysis resulted in a mean value of approximately 0.95. In the absence of a new method for generating a prior for steepness the default approach reverts to the previously endorsed method, the 2017 prior for steepness ($h$; beta distribution with $\mu$=`r round(model$parameters[model$parameters$Label == "SR_BH_steep","Value"],2)` and $\sigma$=0.15) is retained.  

<!--chapter:end:22biology.Rmd-->

\clearpage
# Figures

<!-- ====================================================================== --> 
<!-- ******************* Data Used in the Model *************************** --> 
<!-- ====================================================================== --> 


```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(base_loc, "plots", "data_plot2.png"), 
caption = "Summary of data sources used in the base model",
label = 'data-plot')
```	


<!-- ====================================================================== -->  
<!-- ****************** Catches Used in the Model ************************* --> 
<!-- ====================================================================== -->  

```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "catch2 landings stacked.png"), 
caption = "Catches by fleet used in the base model",
label = 'catch')
```	

<!-- ======================================================================= -->
<!-- **********************  Length Samples ******************************** --> 
<!-- ======================================================================= -->


```{r, results = 'asis'}
#add in length comp plots with captions and labels
#fleets with length

for (i in unique(model$len_comp_fit_table$Fleet)) {
  add_figure(
    filein = dir(paste0(base_loc, "/plots"), 
                 pattern = paste0("comp_lendat_bubflt", i, "mkt0.*\\.png$"), full.names = TRUE),
    caption = paste0("Length composition data from the ", model$FleetNames_parsed[i]),
    label = paste0("len-data-", model$FleetNames_label[i]),
    width = 50,
    height = 50
  )

   add_figure(
    filein = dir(paste0(base_loc, "/plots"), 
                 pattern = paste0("comp_lendat_data_weighting_TA1.8_",
                                  model$FleetNames[i], ".png$"), full.names = TRUE),
    caption = paste0("Mean length for the ", 
                     model$FleetNames_parsed[i], " with 95 percent confidence intervals"),
    label = paste0('mean-com-len-data-', model$FleetNames_label[i]),
    width = 50,
    height = 50
  )

}
```
<!-- ======================================================================= -->
<!-- *************************     Biology     ***************************** --> 
<!-- ======================================================================= -->


```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(fig_loc, "oldfish.jpg"), 
caption = "Photograph of the *oldest* aged fish used in the assessment with annuli marked by B. Kamikawa (NWFSC).",
label = 'oldfish')
```

```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(fig_loc, "Reader 1 vs Reader 2.png"), 
caption = "Aging precision between initial and blind double reads for vermilion. 
Numbers in the bubbles are the sample sizes of otoliths cross-read.",
label = 'reader1reader2')
```	

```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(fig_loc, "True vs Reads (by reader).png"), 
caption = "True versus predicted age for two current age readers at the NWFSC 
from the ageing error software with unbiased reads for reader 1 and curvilinear 
bias for reader 1 and curvilinear standard deviation for both readers.",
label = 'truereads')
```	


<!-- ====================================================================== -->
<!-- *********************    Selectivity            ********************** --> 
<!-- ====================================================================== -->

```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(paste0(base_loc, "/plots"), "sel01_multiple_fleets_length1.png"), 
caption = "Selectivity at length by fleet",
label = 'selex')
```


<!-- ======================================================================= -->
<!-- ********************* Model Diagnostics ******************************* --> 
<!-- ======================================================================= -->


<!-- ======================================================================= -->
<!-- ********************** Fit to the Length Data ************************* --> 
<!-- ======================================================================= -->

```{r, results = 'asis'}
#add in length comp plots with captions and labels
#fleets with length
#Fleet 7 rec_pc_discard doesn't have lengths in either model
unique(model$len_comp_fit_table$Fleet)

for (i in unique(model$len_comp_fit_table$Fleet)) {
  add_figure(
    filein = dir(paste0(base_loc, "/plots"), 
                 pattern = paste0("comp_lenfit_residsflt", i, "mkt0.*\\.png$"), full.names = TRUE),
    caption = paste0("Pearson residuals for the ", model$FleetNames_parsed[i], ". Closed 
                     bubbles are positive residuals (observed > expected) and open bubbles 
                     are negative residuals (observed < expected)"),
    label = paste0("len-pearson-", model$FleetNames_label[i]),
    width = 50,
    height = 50
  )

   add_figure(
    filein = dir(paste0(base_loc, "/plots"), 
                 pattern = paste0("comp_lenfit_data_weighting_TA1.8_",
                                  model$FleetNames[i], ".png$"), full.names = TRUE),
    caption = paste0("Mean lengths for the ", 
                     model$FleetNames_parsed[i], " with 95 percent confidence 
                     intervals based on current samples sizes"),
    label = paste0('mean-len-fit-', model$FleetNames_label[i]),
    width = 50,
    height = 50
  )

}
```


<!-- ======================================================================= -->
<!-- **************************** Time Series ****************************** --> 
<!-- ======================================================================= -->

```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "ts7_Spawning_output_with_95_asymptotic_intervals_intervals.png"), 
caption = "Estimated time series of spawning output",
label = 'ssb')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "ts1_Total_biomass_(mt).png"), 
caption = "Estimated time series of total biomass",
label = 'tot-bio')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "ts9_Relative_spawning_output_intervals.png"), 
caption = "Estimated time series of relative spawning output",
label = 'depl')
```




<!-- ======================================================================= -->
<!-- ************************** Recruitment ******************************** --> 
<!-- ======================================================================= -->


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "ts11_Age-0_recruits_(1000s)_with_95_asymptotic_intervals.png"), 
caption = "Estimated time series of age-0 recruits (1000s)",
label = 'recruits')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "recdevs2_withbars.png"), 
caption = "Estimated time series of recruitment deviations",
label = 'rec-devs')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "SR_curve.png"), 
caption = "Stock-recruit curve. Point colors indicate year, with warmer colors indicating earlier years and cooler colors in showing later years",
label = 'bh-curve')
```

<!-- ======================================================================= -->
<!-- *********************** Reference Points ****************************** --> 
<!-- ======================================================================= -->


```{r, results = 'asis'}
sa4ss::add_figure(
filein = file.path(base_loc, "plots", "SPR2_minusSPRseries.png"), 
caption = "Estimated 1 - relative spawning ratio (SPR) by year",
label = '1-spr')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "SPR4_phase.png"), 
caption = "Phase plot of the relative biomass (also referred to as fraction unfished) versus the SPR ratio where each point represents the biomass ratio at the start of the year and the relative fishing intensity in that same year. Lines through the final point show the 95 percent intervals based on the asymptotic uncertainty for each dimension. The shaded ellipse is a 95 percent region which accounts for the estimated correlations between the biomass ratio and SPR ratio",
label = 'phase')
```


```{r, results = 'asis'}
add_figure(
filein = file.path(base_loc, "plots", "yield2_yield_curve_with_refpoints.png"), 
caption = "Equilibrium yield curve for the base case model. Values are based on the 2020
fishery selectivity and with steepness fixed at 0.72",
label = 'yield')
```



<!-- ======================================================================= -->
<!-- **************************   Sensitivity  ***************************** --> 
<!-- ======================================================================= -->





<!-- ======================================================================= -->
<!-- ************************** Likelihood Profile ************************* --> 
<!-- ======================================================================= -->






<!-- ======================================================================= -->
<!-- *******************     Retrospectives    ***************************** --> 
<!-- ======================================================================= -->







\newpage



<!--chapter:end:53figures.Rmd-->

\clearpage

# Appendix


```{r, results = "asis"}
files_intro <- dir("./indices", pattern = paste0("*", model.area,".md$"), full.names = TRUE)
ignore <- mapply(read_child, files_intro)
```

<!--chapter:end:54a-appendix-indices.Rmd-->

